### PR TITLE
Refactor Javascript and add Rollup bundler

### PR DESCRIPTION
**Pull request by hadriensevel**
### WIP

### CODE REVIEW

File: website/js/src/statistics.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+document.addEventListener('DOMContentLoaded', async () => {
+    // Load the data from the csv files
+    const trainData = await d3.csv(TRAIN_DATA_CSV_PATH);
+    const stopData = await d3.csv(STOP_DATA_CSV_PATH);
+
+    // Generate the best and worst trains
+    generateBestWorst(
+        trainData,
+        'median_arrival_delay',
```
Comment (giacomoorsi) [inline website/js/src/statistics.js:84]: worst trains are computed with the median and worst stops with the average?

Comment (hadriensevel) [reply website/js/src/statistics.js:84]: Well I'm not really the one to ask for this hahaha, that's the way it was in the original file

Comment (giacomoorsi) [reply website/js/src/statistics.js:84]: Ok, can we make both with the median delay?

Comment (hadriensevel) [reply website/js/src/statistics.js:84]: Ok, but is it in the dataset for the stops?

Comment (hadriensevel) [reply website/js/src/statistics.js:84]: I checked and it is so I'm going to change

**Pull request closed without merging**